src/renderers/webgpu/utils/WebGPUPipelineUtils.js

```diff
@@ -88,7 +88,7 @@ class WebGPUPipelineUtils {
 
 		let blending;
 
-		if ( material.transparent === true && material.blending !== NoBlending ) {
+		if ( material.blending !== NoBlending && ( material.blending !== NormalBlending || material.transparent === true ) ) {
```

Comment:
@rkreis-v @sunag @Mugen87 

As I mentioned in the discussion, to replicate the `WebGLRenderer` logic exactly, I think it should be:

```js
if ( material.blending !== NoBlending && ( material.blending !== NormalBlending || material.transparent !== false ) ) {
```

Reply:
Updated: https://github.com/mrdoob/three.js/commit/4e3db421aef677a668932b221b7dc2fc90ef1dd8